typescript/packages/reality-data-client/package-lock.json

```diff
@@ -0,0 +1,8145 @@
+{
```

Comment:
delete this file

you cant just copy everything over from the other repo into here
you need to delete all bechabll and etc stuff from here or move it to the root
there should be only one set of config